src/identity/interaction/util/BasicIssuerReferenceWebIdOwnershipValidator.ts

```diff
@@ -0,0 +1,48 @@
+import { namedNode } from '@rdfjs/dataset';
+import fetch from '@rdfjs/fetch';
+import type { DatasetResponse } from '@rdfjs/fetch-lite';
+import type { AnyPointer } from 'clownface';
+import clownface from 'clownface';
+import type { DatasetCore } from 'rdf-js';
+import { WebIdOwnershipValidator } from './WebIdOwnershipValidator';
+
+const SOLID_ISSUER = namedNode('http://www.w3.org/ns/solid/terms#oidcIssuer');
+
+/**
+ * Validates is a WebId is okay to register based on if it
+ * references this as an issuer.
+ *
+ * Note: This file uses clownface because I like clownface.
+ * If needed, we can refactor it to use whatever rdf library
+ * Ruben prefers.
+ */
+export class BasicIssuerReferenceWebIdOwnershipValidator extends WebIdOwnershipValidator {
+  private readonly issuer: string;
+
+  public constructor(issuer: string) {
+    super();
+    this.issuer = issuer;
+  }
+
+  public async assertWebId(webId: string): Promise<void> {
+    let rawResponse: DatasetResponse<DatasetCore>;
+    try {
+      rawResponse = await fetch(webId) as DatasetResponse<DatasetCore>;
+    } catch {
+      throw new Error('Cannot fetch WebId');
+    }
+    let dataset: AnyPointer;
+    try {
+      dataset = clownface({
+        dataset: await rawResponse.dataset(),
+      });
```

Comment:
See our existing `src/storage/conversion/RdfToQuadConverter.ts`

Reply:
Using /conversion/RdfToQuadConverter would require me to first turn the fetch result into a `representation` and create an `identifier`, then parse a `basicRepresentation`. All of these makes sense in the LDP, but in the IdP doesn't care about representations. You would also need to inject the representation upon config. It just seems like added complexity.